### DISCUSSION TITLE
1panel 显示服务内部错误: database disk image is malformed (11)

### DISCUSSION THREAD

**Discussion by jchen9381** [1Panel-dev/1Panel#5047] [Q&A]
![image](https://github.com/1Panel-dev/1Panel/assets/77962199/08d3d5dc-477e-46b6-9adc-b0cd91d80c5f)
内存和空间都是够的，然后文件应用商城都显示不出来。有没有大佬知道咋解决

**Comment by wanghe-fit2cloud** [comment]
可以先尝试使用 `1pctl restore` 命令回滚至上一个稳定版本，然后再升级系统试一下。

**Reply by Wuqiyang312** [reply]
遇到相同问题，由于没有上个稳定版本，我选择升级，但是升级后问题依旧。这边有更加详细的排除方法吗？
